short-read-mngs/idseq_utils/idseq_utils/batch_run_helpers.py

```diff
@@ -1,77 +1,106 @@
-    job_id, job_queue, job_definition, chunk_id, status, alignment_algorithm
-):
-    log.info(
-        "alignment_batch_job_status",
-        extra={
-            "job_id": job_id,
-            "chunk_id": chunk_id,
-            "job_queue": job_queue,
-            "job_definition": job_definition,
-            "status": status,
-            "alignment_algorithm": alignment_algorithm,
-        },
-    )
+def _bucket_and_key(s3_path: str):
+    parsed_url = urlparse(s3_path, allow_fragments=False)
+    return parsed_url.netloc, parsed_url.path.lstrip("/")
 
 
 def _get_job_status(job_id):
-    batch_job_desc_bucket = boto3.resource("s3").Bucket(get_batch_job_desc_bucket())
+    batch_job_desc_bucket = boto3.resource("s3").Bucket(f"aegea-batch-jobs-{account_id}")
     key = f"job_descriptions/{job_id}"
     try:
         job_desc_object = batch_job_desc_bucket.Object(key)
         return json.loads(job_desc_object.get()["Body"].read())["status"]
     except ClientError as e:
         if e.response["Error"]["Code"] == "NoSuchKey":
             # Warn that the object is missing so any issue with the s3 mechanism can be identified
-            log.debug("missing_job_description_ojbect", extra={key: key})
+            log.debug(f"missing_job_description_ojbect key: {key}")
             # Return submitted because a missing job status probably means it hasn't been added yet
             return "SUBMITTED"
         else:
             raise e
 
 
-def _run_batch_job(job_name, job_queue, job_definition, environment, chunk_id, alignment_algorithm, retries):
-    client = boto3.client("batch")
-    response = client.submit_job(
+def _run_batch_job(
+    job_name: str,
+    job_queue: str,
+    job_definition: str,
+    environment: Dict[str, str],
+    retries: int,
+):
+    response = _batch_client.submit_job(
         jobName=job_name,
         jobQueue=job_queue,
         jobDefinition=job_definition,
         containerOverrides={
-            "environment": environment,
+            "environment": [{
+                "name": k,
+                "value": v,
+            } for k, v in environment.items()],
+            # (524288 - 1024) / 2, 524288 = r5d.24xlarge memory
+            #   2 = jobs per instance, 1024 = leftover for other processes
+            "memory": 261632,
+            "vcpus": 48,  # 96 / 2, 96 = r5d.24xlarge vcpus , 2 = jobs per instance
         },
         retryStrategy={"attempts": retries},
     )
     job_id = response["jobId"]
-    _log_alignment_batch_job_status(
-        job_id, job_queue, job_definition, chunk_id, "SUBMITTED", alignment_algorithm
-    )
+
+    def _log_status(status: str):
+        level = logging.INFO if status != "FAILED" else logging.ERROR
+        log.log(
+            level,
+            "batch_job_status " + json.dumps({
+                "job_id": job_id,
+                "job_name": job_name,
+                "job_queue": job_queue,
+                "job_definition": job_definition,
```

Comment:
It might be good to print the environment, since failures often come from incorrect environment inputs